packages/server/src/plugin/usageReporting/options.ts

```diff
@@ -14,23 +14,63 @@ export interface ApolloServerPluginUsageReportingOptions<
 > {
   //#region Configure exactly which data should be sent to Apollo.
   /**
-   * By default, Apollo Server does not send the values of any GraphQL variables to Apollo's servers, because variable
-   * values often contain the private data of your app's users. If you'd like variable values to be included in traces, set this option.
-   * This option can take several forms:
+   * Apollo Server's usage reports describe each individual request in one of
+   * two ways: as a "trace" (a detailed description of the specific request,
+   * including a query plan and resolver tree with timings and errors, as well
+   * as optional details like variable values and HTTP headers), or as part of
+   * aggregated "stats" (where invocations of the same operation from the same
+   * client program are aggregated together rather than described individually).
+   * Apollo Server uses an heuristic to decide which operations to describe as
+   * traces and which to aggregate as stats.
+   *
+   * By setting the `sendTraces` option to `false`, Apollo Server will describe
+   * *all* operations as stats; individual requests will never be broken out
+   * into separate traces. If you set `sendTraces: false`, the Traces view in
```

Comment:
How would you feel about rephrasing the second sentence to something like this:
```suggestion
   * into separate traces. If you set `sendTraces: false`, then Apollo Studio's Traces view won't show any traces ( other Studio functionality will be unaffected). 
```